### DISCUSSION TITLE
[Bug report] Hero Image problem with locked width

### DISCUSSION THREAD

**Discussion by YerongAI** [vuepress/core#1344] [Q&A]
### Description

When I set an image for my homepage, the image height was locked to 280.

![image](https://github.com/vuepress/vuepress-next/assets/44803668/73ff9998-47d0-43a4-9f56-2257e47b64aa)

I don't know what is the height attri from. And I don't have set any extra settings. Just init my project and put a hero image here.

### Reproduction

Just use v2.vuepress.vuejs.org/new to open a new project.

### Used Package Manager

yarn

### System Info

```shell
I don't know how to do that, but maybe the info above is enough?
```

**Comment by Mister-Hope** [comment]
So why do you think this is a bug?

**Reply by YerongAI** [reply]
If I have an image with size 140×140, the image will be double enlarge. It should not be resized.
